crypto/asn1/x_long.c

```diff
@@ -56,11 +56,30 @@ static void long_free(ASN1_VALUE **pval, const ASN1_ITEM *it)
     *(long *)pval = it->size;
 }
 
+static int num_bits_ulong(unsigned long value)
```

Comment:
Committed. As for "good reason to recreate a similar situation".  I couldn't help thinking what's up with x_long.c module *itself* all along? I mean how is it being dependent on sizeof(long) useful in interoperable world? ...